Model/Data/Email.php

```diff
@@ -0,0 +1,136 @@
+<?php
+namespace Omise\Payment\Model\Data;
+
+class Email
+{
+    /**
+    * @var \Magento\Framework\App\Config\ScopeConfigInterface
+    */
+    protected $_scopeConfig;
+
+    /**
+     * @var \Omise\Payment\Model\Api\Charge
+     */
+    protected $_charge;
+
+    /**
+     * @var \Magento\Framework\View\Asset\Repository
+     */
+    protected $_assetRepo;
+
+    /**
+    * @var  \Magento\Framework\Mail\Template\TransportBuilder
+    */
+    private $_transportBuilder;
+
+    /**
+     * @var \Magento\Store\Model\StoreManagerInterface
+     */
+    protected $_storeManager;
+
+    /**
+     * @var \Omise\Payment\Helper\OmiseHelper
+     */
+    protected $_helper;
+
+    /**
+     * @var string
+     */
+    private $storeName;
+
+    /**
+     * @var string
+     */
+    private $emailTemplate;
```

Comment:
just wonder if we have any naming convention for the class' property? (saw some have prefix `_`)

Reply:
Just found that, according to PSR-2 we shouldn't be using `_` in the naming conventions for class properties. There are some exceptional cases. Probably, we need to remove `_` from all of the possible class properties throughout the plugin. I am planning to do this in future PR which would contains all changes related PHPCS,PSR-2.